src/util/jira.ts

```diff
@@ -0,0 +1,26 @@
+import { JiraAuthor } from "../interfaces/jira";
+
+export const getJiraAuthor = (author?: Author): JiraAuthor => author ? {
+	avatar: author.avatar_url || author.avatarUrl || `https://github.com/${author.login}.png`,
+	name: author.name || author.login || "unknown",
+	email: author.email || `${author.login || "unknown"}@noreply.user.github.com`,
+	url: author.url || author.html_url || author.user?.url || `https://github.com/${author.login}`
+} : {
+	avatar: "https://github.com/ghost.png",
+	name: "Deleted User",
+	email: "deleted@noreply.user.github.com",
+	url: "https://github.com/ghost"
+};
+
+interface Author {
+	avatar_url?: string;
+	avatarUrl?: string;
```

Comment:
Let's add a comment with clarification why there are both `avatar_url` and `avatarUrl`